token/program-2022/src/instruction.rs

```diff
@@ -409,6 +427,125 @@ pub enum TokenInstruction {
         /// The freeze authority/multisignature of the mint.
         freeze_authority: COption<Pubkey>,
     },
+    /// Initialize a new mint for payment systems.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   0. `[writable]` The mint to initialize.
+    InitializePaymentMint {
+        /// Number of base 10 digits to the right of the decimal place.
+        decimals: u8,
+        /// The authority/multisignature to mint tokens.
+        mint_authority: Pubkey,
+        /// The freeze authority/multisignature of the mint.
+        freeze_authority: COption<Pubkey>,
+        /// Pubkey that must own any provided fee account
+        fee_account_owner: Pubkey,
```

Comment:
Both  `fee_config_authority` and `fee_withdraw_authority` are set to this pubkey?  Also `COption<Pubkey>` I think

Reply:
Bah thanks, missed that one. And I was thinking it would be weird to create a mint with transfer fees and not set the authorities, since they wouldn't be settable after that. There is something cool to having no config authority and a harvest authority, since then you know the fee will never change.  Works for me!